diagram-viewer/src/main/java/com/powsybl/diagram/viewer/MainViewController.java

```diff
@@ -152,6 +156,37 @@ public TreeItem<Container<?>> fromString(String string) {
         sldViewController.addListener((observable, oldValue, newValue) -> updateSldDiagrams());
     }
 
+    private void initializeNetworkFactories() {
+        Map<String, Supplier<Network>> suppliers = new HashMap<>();
+        suppliers.put("BatteryNetwork", com.powsybl.iidm.network.test.BatteryNetworkFactory::create);
```

Comment:
Could you add `import com.powsybl.iidm.network.test.*;` so that these lines are more compact?